app/views/api/v1/sessions/create_success.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+json.id @resource.id
```

Comment:
Just a little thing here, you can align all the `@resource...` to make it more readable.